app/views/application/_navbar.haml

```diff
@@ -10,14 +10,14 @@
             New event
         %li
           %p.navbar-text
+            = image_tag current_user.image, size: '20x20', alt: "Profile image"
             = current_user.name
-            
-            = link_to logout_path do
-              %i.fa.fa-sign-out
-              Sign out
+        %li
+          = link_to logout_path do
+            %i.fa.fa-sign-out
```

Comment:
do we have some helper method for creating fontawesome icon provided by gem? like `fa_icon('sign-out')`? if not please create this kind of helper and use it in navbar :)